### PR TITLE
Feature/ab#77 use the same dialogs for delete confirmation

### PR DESCRIPTION
**Pull request by DerCed**
## Description
Same dialogs for both siuations

## Motivation and Context
All should look the same

## How has this been tested?
Tested AppTemplate delete

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: webclient/app/src/app/commons/alert/ConfirmationDialog.jsx [hunk 1]
```diff
@@ -1,32 +1,60 @@
...
+            .catch(() => {
+                setIsProcessing(false)
+            })
+    }
 
     return (
         <Dialog
             open={open}
             onClose={onClose}
-            aria-labelledby="alert-dialog-title"
```
Comment (witchpou) [inline webclient/app/src/app/commons/alert/ConfirmationDialog.jsx:17]: can we please use this kind of text:

image

File: webclient/app/src/app/commons/alert/ConfirmationDialog.jsx [hunk 2]
```diff
@@ -1,32 +1,60 @@
...
+                <DialogContentText id="confirm-dialog-description">
+                    {message}
+                </DialogContentText>
+            </DialogContent>
             <DialogActions>
-                <Button onClick={onClose}>{t("button.no")}</Button>
-                <Button onClick={onSubmit} autoFocus>
-                    {t("button.yes")}
-                </Button>
+                <Button onClick={onClose}>{t("button.cancel")}</Button>
```
Comment (witchpou) [inline webclient/app/src/app/commons/alert/ConfirmationDialog.jsx:44]: Did you test the cancel button? It was not working on my side in the old code.

File: webclient/app/src/app/features/apps/appCard/AppCard.jsx [hunk 3]
```diff
@@ -1,49 +1,33 @@
...
+    Grid,
     IconButton,
     Snackbar,
-    Grid,
-    Typography,
-    CardActions,
-    Divider
+    Typography
 } from "@mui/material";
-import { Delete, Edit, MoreHoriz } from "@mui/icons-material";
```
Comment (witchpou) [inline webclient/app/src/app/features/apps/appCard/AppCard.jsx:19]: my IDE is using blanks...

File: webclient/app/src/app/features/apps/appCard/AppCard.jsx [hunk 4]
```diff
@@ -55,20 +39,21 @@ function AppCard(props) {
...
                 </CardContent>
-                <Divider />
+                <Divider/>
                 <CardActionArea onClick={() => history.push("/apps/" + app.id)}>
                     <CardContent>
                         <Typography variant="body2" className={appCardStyles.description}>
                             {app.packageName}
                         </Typography>
                         <CardActions className={appCardStyles.cardActions}>
-                            <MoreHoriz className={appCardStyles.cardActions} color="primary" />
```
Comment (witchpou) [inline webclient/app/src/app/features/apps/appCard/AppCard.jsx:71]: Why this long-time formatting standard has changed?

**Review by witchpou — CHANGES_REQUESTED** [review]

File: webclient/app/src/app/commons/alert/ConfirmationDialog.jsx (hunk 1 above)
Comment (DerCed) [reply webclient/app/src/app/commons/alert/ConfirmationDialog.jsx:17]: This styling is not very Material UI compliant and in my opinion quite confusing.
If you insist, I can revert that.
https://material.io/components/dialogs#alert-dialog

File: webclient/app/src/app/commons/alert/ConfirmationDialog.jsx (hunk 2 above)
Comment (DerCed) [reply webclient/app/src/app/commons/alert/ConfirmationDialog.jsx:44]: Works on my side

File: webclient/app/src/app/features/apps/appCard/AppCard.jsx (hunk 4 above)
Comment (DerCed) [reply webclient/app/src/app/features/apps/appCard/AppCard.jsx:71]: My IDE does this formatting forever...

File: webclient/app/src/localization/translations-de-DE.js [hunk 5]
```diff
@@ -31,10 +31,10 @@ const translationsDeDE = {
     "apptemplates.loading": "die Templates werden geladen",
     "apptemplates.loadingError": "Die Templates konnten nicht geladen werden.",
 
-    "app.loading": "App wird geladen",
+    "app.loading": "Apps wird geladen",
     "app.LoadingError": "Es ist ein Fehler beim Laden aufgetreten.",
     "app.doesNotExist": "Die angegebene App exisitert nicht.",
-    "app.delete.title": "App löschen",
+    "app.delete.title": "Wollen Sie diese App wirklich löschen?",
```
Comment (witchpou) [inline webclient/app/src/localization/translations-de-DE.js:37]:
```suggestion
    "app.delete.title": "App löschen",
```

File: webclient/app/src/localization/translations-de-DE.js [hunk 6]
```diff
@@ -31,10 +31,10 @@ const translationsDeDE = {
...
     "app.delete.message": "Diese Operation kann nicht rückgängig gemacht werden.",
```
Comment (witchpou) [inline webclient/app/src/localization/translations-de-DE.js:38]:
```suggestion
    "app.delete.message": "Wollen Sie diese App wirklich löschen?",
```

File: webclient/app/src/localization/translations-en-EN.js [hunk 7]
```diff
@@ -34,7 +34,7 @@ const translationsEnEN = {
     "app.loading": "app is loading",
     "app.LoadingError": "An error occurred while loading the app.",
     "app.doesNotExist": "The provided App does not exist",
-    "app.delete.title": "Delete app",
+    "app.delete.title": "Are you sure to delete this App?",
     "app.delete.message": "This operation cannot be reversed.",
```
Comment (DerCed) [inline webclient/app/src/localization/translations-en-EN.js:38]:
```suggestion
    "app.delete.message": "Do you want to delete the App?",
```

File: webclient/app/src/localization/translations-en-EN.js [hunk 8]
```diff
@@ -85,8 +85,8 @@ const translationsEnEN = {
     "relationship.targetField.hint": "Name of field with type {{entityName}}. Upper and lower case letters are allowed with numbers. First character must be a lower case letter.",
     "relationship.targetEntity.empty": "not available",
 
-    "appTemplate.delete.title": "Delete Template",
-    "appTemplate.delete.message": "Do you want to delete the template?",
+    "appTemplate.delete.title": "Are you sure to delete this App Template?",
+    "appTemplate.delete.message": "This operation cannot be reversed.",
```
Comment (DerCed) [inline webclient/app/src/localization/translations-en-EN.js:89]:
```suggestion
    "appTemplate.delete.message": "Do you want to delete the template?",
```

**Review by witchpou — CHANGES_REQUESTED** [review]

**Review by witchpou — APPROVED** [review]

**Pull request merged**